src/YamahaAPI.ts

```diff
@@ -67,12 +67,12 @@ interface Response {
 export interface StatusResponse {
     response_code: number;
     power: string;
-    //sleep: number;
+    sleep: number;
     volume: number;
     mute: boolean;
     max_volume: number;
     input: string;
-    input_text: string;
+    input_text?: string;
```

Comment:
probably that was not intended to be in this PR?
```suggestion
    input_text: string;
```

Reply:
This is what I see when I'm doing this call for R-N602, so I decided to left it